Framework/Core/include/Framework/AnalysisTask.h

```diff
@@ -21,6 +21,7 @@
 #include "Framework/Expressions.h"
 #include "Framework/ExpressionHelpers.h"
 #include "Framework/EndOfStreamContext.h"
+#include "Framework/GroupSlicer.h"
```

Comment:
Do you really need to include it by default? Can't we include it only when needed?

Reply:
It is used in this file, for grouping for analysis task.